Automattic/jetpack
# VideoPress: Pull and show title and description of VideoPress block

In this PR:

* creates a hook to pull the video item data
* sanitize the data (decodeEntities)
* render the title and description in the DetailsControl when it's ready

Fixes https://github.com/Automattic/jetpack/issues/26191

#### Changes proposed in this Pull Request:

* Jetpack: pull and show title and description in the VideoPress Video sidebar

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:







* Upload a video (VideoPress)
* Edit `title` and `description` from the media gallery (wp-admin) or using Calypso
* Once the video data saves, do a hard refresh in the block editor
* Confirm the title and description have the proper data

wp-admin | Calypso | Block Editor
-----|----|----
<img width="890" alt="Screen Shot 2022-09-13 at 13 34 31" src="https://user-images.githubusercontent.com/77539/189957667-5f915240-4c61-48ed-a074-b87e879cff9c.png"> | <img width="938" alt="Screen Shot 2022-09-13 at 13 34 55" src="https://user-images.githubusercontent.com/77539/189957663-d4f2c791-248c-4be1-9aa2-f6c4677ee3f9.png"> | <img width="923" alt="Screen Shot 2022-09-13 at 13 35 19" src="https://user-images.githubusercontent.com/77539/189957655-39e43e57-fd80-47e3-a3bd-23d86a9d3050.png">
